### PR TITLE
Changes for beneficiary admin pages

### PR DESCRIPTION
**Pull request by vitaliel**
### What does it fix?

Creating a beneficiary from client app.

Main changes this PR brings:
* Get Requests filtered by beneficiary.
* CRUD operations for beneficiary
* Update Beneficiary, Request models with new fields.
* More Db seeds for Beneficiary, Request, Cluster models.
* test factories for Request, Cluster, Volunteer models.

### How has it been tested?

Using client js app from PR: https://github.com/code4moldova/voluntar-web/pull/137

### CODE REVIEW

File: backend/models/beneficiary_model.py [hunk 1]
```diff
@@ -18,52 +17,59 @@
...
-    landline = IntField(min_value=16, max_value=120)
+    phone = StringField()
+    landline = StringField()
     age = IntField(min_value=16, max_value=120)
-    zone = StringField(choices=[zone.value for zone in Zone], required=True)
+    zone = StringField(choices=[zone.name for zone in Zone], required=True)
     address = StringField(max_length=500, required=True)
-    special_condition = StringField(choices=[sc.value for sc in SpecialCondition], default="")
+    apartament = StringField(max_length=10)
+    scara = StringField(max_length=10)
```
Comment (johnsmithm) [inline backend/models/beneficiary_model.py:26]: it is a good idea to use English for all fields name, maybe someone from Latvia will use our code :)

File: backend/models/beneficiary_model.py [hunk 2]
```diff
@@ -18,52 +17,59 @@
 class Beneficiary(Document):
     first_name = StringField(max_length=500)
     last_name = StringField(max_length=500)
-    phone = IntField(min_value=16, max_value=120)
-    landline = IntField(min_value=16, max_value=120)
+    phone = StringField()
+    landline = StringField()
     age = IntField(min_value=16, max_value=120)
-    zone = StringField(choices=[zone.value for zone in Zone], required=True)
+    zone = StringField(choices=[zone.name for zone in Zone], required=True)
```
Comment (alexciornii) [inline backend/models/beneficiary_model.py:23]: why you did change it?

File: backend/models/beneficiary_model.py (hunk 1 above)
Comment (vitaliel) [reply backend/models/beneficiary_model.py:26]: I will rename, just moved the fields from the bottom.

File: backend/models/beneficiary_model.py (hunk 2 above)
Comment (vitaliel) [reply backend/models/beneficiary_model.py:23]: We want to use field name of enum, not label. https://github.com/code4moldova/voluntar-web/pull/137/files#diff-7c4276a86b2d0c9995ca3d396372bd9fR4

**Review by johnsmithm — APPROVED** [review]

**Pull request merged**